config/externalnamenottested.go

```diff
@@ -394,4 +398,164 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"aws_elasticsearch_domain_policy": config.IdentifierFromProvider,
 	// Elasticsearch domains can be imported using the domain_name
 	"aws_elasticsearch_domain_saml_options": config.ParameterAsIdentifier("domain_name"),
+
+	// pinpoint
+	//
+	// Pinpoint ADM Channel can be imported using the application-id
+	"aws_pinpoint_adm_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint APNs Channel can be imported using the application-id
+	"aws_pinpoint_apns_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint APNs Sandbox Channel can be imported using the application-id
+	"aws_pinpoint_apns_sandbox_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint APNs VoIP Channel can be imported using the application-id
+	"aws_pinpoint_apns_voip_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint APNs VoIP Sandbox Channel can be imported using the application-id
+	"aws_pinpoint_apns_voip_sandbox_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint App can be imported using the application-id
+	"aws_pinpoint_app": config.IdentifierFromProvider,
+	// Pinpoint Baidu Channel can be imported using the application-id
+	"aws_pinpoint_baidu_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint Email Channel can be imported using the application-id
+	"aws_pinpoint_email_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint Event Stream can be imported using the application-id
+	"aws_pinpoint_event_stream": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint GCM Channel can be imported using the application-id
+	"aws_pinpoint_gcm_channel": FormattedIdentifierFromProvider("", "application_id"),
+	// Pinpoint SMS Channel can be imported using the application-id
+	"aws_pinpoint_sms_channel": FormattedIdentifierFromProvider("", "application_id"),
+
+	// qldb
+	//
+	// QLDB Ledgers can be imported using the name
+	"aws_qldb_ledger": config.NameAsIdentifier,
+	// No import
```

Comment:
Thanks @sergenyalcin for taking note of these TODOs. Let's do our best to enforce these during testing & reviews.